lib/consts.php

```diff
@@ -24,6 +24,7 @@
   define("MAIL_USER_NEWSLETTER_TOKEN", "{%USER_NEWSLETTER_TOKEN}");
   define("MAIL_USER_VERIFY_TOKEN",     "{%USER_VERIFY_TOKEN}");
   define("MAIL_WEBSITE",               "{%WEBSITE}");
+  define("MAIL_REFERENCES",            "{%INDUSTRY_REFERENCES}");
```

Comment:
Why do use the name _INDUSTRY_REFERENCES_ at some points and just _REFERENCES_ at other points? IMHO it would be less error-prone to choose one naming and stick to it throughout.